includes/templates/single-web-story.php

```diff
@@ -46,6 +49,7 @@
 			object-fit: cover;
 		}
 	</style>
+	<script type="application/ld+json"><?php echo wp_json_encode( $metadata, JSON_UNESCAPED_UNICODE ); ?></script>
```

Comment:
This code should be added in via the `do_action( 'web_stories_story_head' );` so it can be unhooked.